app/views/conferences/_table.html.erb

```diff
@@ -1,61 +1,31 @@
-<table class="table-striped table-bordered table-condensed conferences">
-  <thead>
-  <tr>
-    <th> <%= sortable "Conference" %> </th>
-    <th> <%= sortable "Location" %> </th>
-    <th> <%= sortable "Dates" %> </th>
-    <th> Attendees </th>
-    <th> Tickets left </th>
-    <th> Lightning Talk Slots?</th>
-  </tr>
-  </thead>
-
-  <tbody>
-  <% conferences.where(round: 2).each do |conference| %>
-  <tr>
-    <td> <%= link_to conference.name, [namespace, conference] %></td>
-    <td> <%= conference.location %></td>
-    <td> <%= conference.date_range %></td>
-    <td> <%= conference.attendees.any? ? links_to_attendances(conference).join(', ').html_safe : '-' %></td>
-    <td> <%= conference.tickets_left %></td>
-    <td><%= conference.lightningtalkslots? ? "yes" : "no" %></td>
-  </tr>
-      <% end %>
-    </tbody>
-</table>
-
-<br>
-<p><%= link_to "View as Calendar", calendar_path %>
-</p>
-
-<p><%= link_to "Show/Hide First Round Conferences", "#", onclick: "$('.first-conferences').toggle()" %></p>
-
-<div class="first-conferences" style='display:none;'>
-<h1 class=" header">Conferences (1st round)</h1>
-
+<h1 class="header">Conferences</h1>
```

Comment:
Sure - was just wondering if it's on purpose or not.

I'm generally against dropping slim as I think it gives an interesting tiny little "transfer-thinking" effort to the Teams App: HTML is limited and not a programming language. So it just takes a few looks at the docs *(and of course some trial and error)* to express the same things one has in mind for HTML in slim.

But yeah, can imagine it being tedious if you start with a full .erb implementation and try to "convert" it to .slim 😉